components/list_item/default_list_item.vue

```diff
@@ -38,6 +38,9 @@
       <!-- @slot Slot for the right content -->
       <slot name="right" />
     </section>
+    <template v-if="$slots.selected">
+      <slot name="selected" />
+    </template>
```

Comment:
No reason to wrap in a template here

```suggestion
    <slot name="selected" />
```

Reply:
Oops, yep, that should not be there